ui/src/singletons/store.ts

```diff
@@ -220,6 +253,9 @@ const newNode = (eipId: EipId, position: XYPosition) => {
       label: "New Node",
     },
   }
+  const isHorizontal = layout === "horizontal"
+  ;(node.targetPosition = isHorizontal ? Position.Left : Position.Top),
+    (node.sourcePosition = isHorizontal ? Position.Right : Position.Bottom)
```

Comment:
The formatting is a bit off here. Why not include these in the object initialization above?
```
const node: EipFlowNode = {
    id: id,
    type: EIP_NODE_KEY,
    position: position,
    targetPosition: isHorizontal ? Position.Left : Position.Top,
    sourcePosition: isHorizontal ? Position.Right : Position.Bottom,
    data: {
      eipId: eipId,
      label: "New Node",
    },
  }
```